projects/Mallard/src/navigation/navigators/header.tsx

```diff
@@ -13,6 +13,8 @@ import { addStaticRouter } from '../helpers/base'
 
 interface NavigationOptions {
     title?: string
+    showHeaderRight?: boolean
+    hideHeaderLeft?: boolean
```

Comment:
Can these be consistent in naming

Reply:
I've done this to reflect that the button on the left should largely always show, bar when you wish to override it to hide it. Similarly the opposite for show header right which should never show unless you explicitly ask for it.

Reply:
Fixed.